File: sample-conference-videochat/sample-conference-videochat/Components/Settings/VideoSettingsViewController/BaseItemModel.m
//
//  BaseItemModel.m
//  sample-conference-videochat
//
//  Created by Injoit on 30/09/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "BaseItemModel.h"
#import "SettingCell.h"

@implementation BaseItemModel

- (Class)viewClass {
    
    return [SettingCell class];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Settings/BaseSettingsViewController/BaseItemModel.swift
//
//  BaseItemModel.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 22.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class BaseItemModel {
    //MARK: - Properties
    var title = ""
    var data: Any?
    
    //MARK: - Public Methods
    func viewClass() -> AnyClass {
        return  SettingCell.self
    }
}
